bukuserver/server.py

```diff
@@ -37,7 +37,7 @@
 
 def get_tags():
     """get tags."""
-    tags = getattr(flask.g, 'bukudb', BukuDb()).get_tag_all()
+    tags = getattr(flask.g, 'bukudb', BukuDb(current_app.config.get('BUKUSERVER_DB'))).get_tag_all()
```

Comment:
start from here, i recommend create a function to replace a `BukuDb`.

basically something similar like this:

```python
def get_bukudb():
    db_file = current_app.config.get('BUKUSERVER_DB_FILE', None)
	if db_file is None:
		return BukuDb()
	return BukuDb(db_file)
```

it may seem too much but if later `BukuDb changed positional argument or default argument, it will be easier too update.

also when the user feature come it will only changed on this place

Reply:
It doesn' t seem too much, I was just staying close to the initial implementation. I'm happy to add this :-)